pkg/controllers/multicluster/serviceexport_controller.go

```diff
@@ -259,6 +259,8 @@ func (r *ServiceExportReconciler) extractEndpoints(ctx context.Context, svc *v1.
 		return nil, err
 	}
 
+	svcExportCreationTimestamp := svcExport.ObjectMeta.CreationTimestamp.Time.UnixMilli()
```

Comment:
We need to perform nil check before calling `Time.UnixMilli()`. As per the doc

>CreationTimestamp is a timestamp representing the server time when this object was
	// created. It is not guaranteed to be set in happens-before order across separate operations.
	// Clients may not set this value. It is represented in RFC3339 form and is in UTC.
	//
	// Populated by the system.
	// Read-only.
	// Null for lists.
	// More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
	// +optional
	CreationTimestamp Time `json:"creationTimestamp,omitempty" protobuf:"bytes,8,opt,name=creationTimestamp"`

Reply:
Added nil check. Set timestamp to 0 when errors or is not set.